### ISSUE TITLE
Module 'distro' has no '__version__' member

### ISSUE THREAD

**Issue by AdnanHodzic** [python-distro/distro#265]
As it's case with most Python packages, if you run i.e:

```
print("psutil version:", psutil.__version__)
```

you'll end up getting package version, i.e: `psutil: 5.7.2`

However, if I do the same for distro package, i.e:

```
print("distro:", distro.__version__)
```

I'll immediately get `Module 'distro' has no '__version__' member`

Would it be possible to add this functionality in one of the future versions instead of having to resort to hacks/workarounds in order to print distro package version as part of a Python app.

Thanks!

**Comment by jdufresne** [comment]
The modern way to do this is with [importlib.metadata](https://docs.python.org/3/library/importlib.metadata.html). 

```
>>> from importlib.metadata import version
>>> version('distro')
'1.5.0'
```

**Comment by jdufresne** [comment]
Thanks for the report. Resolved in a3da4d454027e591a88039e22879f6f4edb09fb8.
